:bug: Change independent corner radius input tooltips

astudentinearth opened the pull request:
Make the inputs show a tooltip for the relevant corner(e.g. "Top left") instead of "Radius".
Completes #3305 ([Taiga #5486](https://tree.taiga.io/project/penpot/issue/5486))
This pull request does not introduce breaking changes.

After the change the tooltips are shown like this:
![image](https://github.com/penpot/penpot/assets/95227070/35572ea8-8167-4d95-9e46-ff34dc47d5c2)

astudentinearth committed:
:bug: Change independent corner radius input tooltips

Make the inputs show a tooltip for the relevant corner(e.g. "Top left") instead of "Radius"

Signed-off-by: Burak Yeniçeri <burak.yn.dev@gmail.com>

superalex commented:
Hey!, thank you very much @astudentinearth 

As radius and interactions are two different concepts I think it would be a good idea using new translation keys. Even if right now they have the same translation,  if in the future they change it will affect this too.

You could create the new keys for every language and copy the translations from each language.

astudentinearth commented:
@superalex I understand the concerns about translation keys. I will look into it once I'm available.

superalex commented:
Great!, thank you very much :)

astudentinearth committed:
:bug: Create language keys for border radius corners

Create the keys and copy relevant position strings from already existing `workspace-interaction-pos` keys. Update `measures.cljs` to use the new keys.

Signed-off-by: Burak Yeniçeri <burak.yn.dev@gmail.com>

astudentinearth committed:
Merge branch 'penpot:develop' into astudentinearth-change-radius-tooltips

astudentinearth commented:
I have created the new language keys, however I synced my fork branch with `origin/develop` on the web interface, so there is a merge commit in between. Would that cause a problem?

superalex commented:
Thank you very much!, I've just merged manually this :)

astudentinearth commented:
@superalex Thanks a lot. This was my first ever pull request❤️

superalex commented:
> @superalex Thanks a lot. This was my first ever pull requestheart

It is a great honour for us! <3 <3 <3
